src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerClient.kt

```diff
@@ -18,7 +19,7 @@ interface OrderlyServerAPI : PorcelainAPI
 
 class OrderlyServerClient(
         config: Config,
-        client: OkHttpClient = OkHttpClient()
+        client: OkHttpClient = httpClient
```

Comment:
this is unrelated, but the `OkHttpClient` docs say it's best to use a singleton instance of this class throughout the app